lib/profile/index.js

```diff
@@ -48,6 +49,11 @@ function generate (inputs, opts) {
 
   profile.current_basal = basal.basalLookup(inputs.basals);
   profile.basalprofile = inputs.basals;
+  
+  _.forEach(profile.basalprofile, function(basalentry) {
+    basalentry.rate = +(Math.round(basalentry.rate + "e+3")  + "e-3");		
```

Comment:
What does this do?  Is it another way of rounding to three digits after the decimal place?